src/ws/session-utils.c

```diff
@@ -36,6 +36,10 @@
 #include <time.h>
 #include <inttypes.h>
 
+#ifndef _PATH_BTMP
+#define _PATH_BTMP "/var/log/btmp"
```

Comment:
Is that macro defined on any OS already? I don't see it in /usr/include in Fedora at least.

Reply:
I copied this from ssh, I think.  Probably some of the BSDs or something have this define.  Indeed, I don't think any Linux has it.  I'd like to keep it like this, though.